cluster/aws/util.sh

```diff
@@ -1059,7 +1116,9 @@ function kube-up {
     if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
       # Start minions
       start-minions
```

Comment:
Since there is a start minions, why preset does not need to wait for minions?

Reply:
The work has been done inside start minions.

Reply:
Then why the other case need to wait for minions?

Reply:
Other case is waiting for minion machine running after create it. Since my case minion machine was preset and already running, I don't need to wait. And I cannot wait because the check is using aws cli.

Reply:
My point is: start minion and wait for minion seems naturally bind together. If we use start minion but not wait minion, something seems not right. Specifically, feels like there are some logic in start minion that should not be there. Can you provide details in start minion that is needed for preset machines?

Reply:
Discussed. Added comments in code.